docs/user-guide/api-mediation/api-gateway-configuration.md

```diff
@@ -260,3 +259,104 @@ Once Zowe is installed, use the following procedure to limit which components st
+
+Verification of the SAF resource is provided by the following three providers:
+
+- **`endpoint`**
+
+  This is the highest priority provider, such as a REST endpoint call (ZSS or similar one). This option is disabled by default. In Zowe, ZSS has the API to check for SAF   resource authorization.
+  
+- **`native`**
+
+  The Native JZOS classes from Java are used to determine SAF resource access. This is the default provider.
+  
+- **`dummy`**
+
+  This is the lowest priority provider. This is the dummy implementation and is defined in a file.
+
+**Note:** Verification of the SAF resource uses the first available provider based on the specified priority. The default configuration resolves to the native provider. 
+
+You can select a specific provider by specifying the `APIML_SECURITY_AUTHORIZATION_PROVIDER` key in the `instance.env` file. Use the parameter value to
+strictly define a provider. If verification is disabled, select the `endpoint` option. 
+
+**Follow these steps:**
+
+1. Open the file `<Zowe instance directory>/instance.env`.
+2. Add the property `APIML_SECURITY_AUTHORIZATION_PROVIDER` and set desired value.
+3. Restart `Zowe&trade`.
+
+**Examples:**
+```
+APIML_SECURITY_AUTHORIZATION_PROVIDER=endpoint
+```
+```
+APIML_SECURITY_AUTHORIZATION_PROVIDER=native
+```
+```
+APIML_SECURITY_AUTHORIZATION_PROVIDER=dummy
+```
+
+To use the endpoint provider, customize the URL corresponding to the SAF resource authorization. By default, the ZSS API is configured and used. 
+
+**Follow these steps:**
+
+1. Open the file `<Zowe instance directory>/instance.env`.
+2. Modify the property `APIML_SECURITY_AUTHORIZATION_ENDPOINT_URL` and set desired value.
+   The default value for ZSS API is `https://${ZOWE_EXPLORER_HOST}:${GATEWAY_PORT}/zss/api/v1/saf-auth`
+3. Restart `Zowe&trade`.
+
+### Checking providers
+
+#### REST endpoint call
+
+To enable the feature outside of the mainframe, such as when running in Docker, you can use a REST endpoint call using the `GET` method:
+
+- Method: `GET`
+- URL: `{base path}/{userId}/{class}/{entity}/{level}`
+- Response:
+```json5
+    {
+        "authorized": "{true|false}",
+        "error": "{true|false}",
+        "message": "{message}"
+    }
+```
+**Note:** For more information about this REST endpoint call, see [ZSS implementation](https://github.com/zowe/zss/blob/master/c/authService.c).
+
+#### Native
+
+This provider is the easiest approach to use the SAF resource checking feature on the mainframe.
+
+Enable this provider when classes `com.ibm.os390.security.PlatformAccessControl` and `com.ibm.os390.security.PlatformReturned`
+are available on the classpath. This approach uses the following method described in the IBM documentation: [method](https://www.ibm.com/support/knowledgecenter/SSYKE2_8.0.0/com.ibm.java.zsecurity.api.80.doc/com.ibm.os390.security/com/ibm/os390/security/PlatformAccessControl.html?view=kc#checkPermission-java.lang.String-java.lang.String-java.lang.String-int-).
```

Comment:
For clarity, an example of how this method is used could be added here.

Reply:
User does not use it, it's our code that depends on it. Developer doc and samples are documented in our repository. Does that answer your concern?